en/2.1-release-notes.md

```diff
@@ -0,0 +1,277 @@
+Title: 2.1 Release Notes | MAAS
+table_of_contents: True
+
+
+# 2.1 Release Notes
+
+## Important changes
+
+### New MAAS dashboard
+
+In MAAS 2.1, administrators will be shown the new MAAS dashboard after they log
+in to the web UI for the first time. Here, they are prompted to answer a few
+questions in order to get MAAS up and running quickly (see configuration
+journey section for details). In addition, administrators can view hosts that
+have been discovered on the network and quickly convert them to a device in
+MAAS.
+
+### Image streams have been upgraded to v3 (local mirror update required)
+
+In order to support new kernels, MAAS has moved to a new format for image
+streams. Previous releases used stream in 'v2' format. Starting with MAAS 2.1,
+the 'v3' format image stream will be used.
+
+Users upgrading from earlier versions of MAAS who are using the default image
+source will be automatically migrated to the v3 format.
+
+MAAS will not migrate the URL of a non-default image source (i.e. a mirror) as
+it must first contain the v3 stream, which is available at
+`http://images.maas.io/ephemeral-v3/`.
+
+Old images downloaded from the v2 stream will continue to work but the MAAS
+team only supports MAAS 2.1 users using the v3 stream. Note that the default
+image source may eventually drop the v2 stream.
+
+Bootloaders are now included in the default image source (see bootloaders
+section for details). Mirrors should be updated accordingly.
+
+### New HWE and GA kernel naming convention
+
+Starting with MAAS 2.1 and Ubuntu 16.04 LTS (Xenial), MAAS is adhering to a
+new naming convention for hardware enablement (HWE) kernels and general
+availability (GA) kernels. MAAS will no longer support the old naming
+convention for HWE kernels (e.g. hwe-y) when selecting images (Xenial and
+newer) to import. The new nomenclature is as follows:
+
+- `ga-<version>` : The GA kernel has the major kernel version of the kernel which
```

Comment:
Should we remove the spaces before the colons (and throughout the remainder of the doc)?